src/utils/include/utils/sampling.hpp

```diff
@@ -55,7 +55,8 @@ get_cylindrical_sampling_positions(std::pair<double, double> const &r_limits,
   auto const smallest_bin_volume =
       pi() * pow(r_limits.first + delta_r, 2.0) * delta_phi / (2.0 * pi());
```

Comment:
this would be a candidate for `Utils::sqr()`, but it's only called in the constructor of profile observables as far as I can tell, so not performance critical